Jelly/Classes/private/Animators/Animated Animators/CoverAnimator.swift

```diff
@@ -56,6 +57,19 @@ extension CoverAnimator : UIViewControllerAnimatedTransitioning {
         let propertyAnimator = UIViewPropertyAnimator(duration: animationDuration, timingParameters: timingCurveProvider)
 
         propertyAnimator.addAnimations {
+            if isPresentation {
+                if let presentingViewController = transitionContext.viewController(forKey: .from) {
+                    presentingViewController.view.transform = CGAffineTransform(scaleX: self.presentation.depthScale,
+                                                                                y: self.presentation.depthScale)
+                    if UIApplication.shared.delegate?.window??.safeAreaInsets.top ?? 0 > 20 {
+                        let maskLayer = CAShapeLayer()
+                        maskLayer.path = UIBezierPath(roundedRect: presentingViewController.view.bounds, byRoundingCorners: .allCorners, cornerRadii: CGSize(width: 38.5, height: 38.5)).cgPath
```

Comment:
Why did you chose a corner radius of 38.5? 
Wouldnt it be nice if the user could specify its own corner radius for the background scale transformation?